e2e/mv_test.go

```diff
@@ -311,3 +311,51 @@ func TestMoveMultipleS3ObjectsToS3(t *testing.T) {
 		assert.Assert(t, ensureS3Object(s3client, bucket, "dst/"+filename, content))
 	}
 }
+
+// mv s3://bucket/single*.blob* s3://bucket/single/
+func TestMoveWildcardThousandAndTenS3ObjectsToS3(t *testing.T) {
+	t.Parallel()
+
+	bucket := s3BucketFromTestName(t)
+
+	s3client, s5cmd, cleanup := setup(t)
```

Comment:
It took nearly 37seconds to run this test when I forked and run this test locally :) We use `boltdb` as a default backend for e2e tests. But for this test, it is more convenient to use in-memory backend since we are working with a large number of files. 
```suggestion
	s3client, s5cmd, cleanup := setup(t, withS3Backend("mem"))
```

Reply:
wow, awesome, thanks. let me try it :)